code/modules/projectiles/guns/projectile/automatic/ppsh.dm

```diff
@@ -1,98 +1,97 @@
-	name = "\"Ekaterina\" assault SMG"
-	desc = "An experimental sub-machine gun design made for urban combat with a built in silencer and chambered in 9mm.\nThis model of the \"Shpagin\" assault SMG has been reclaimed and repurposed by Nadezhda Marshals, making it free of Excelsior taint."
-	icon = 'icons/obj/guns/projectile/ppsh_bs.dmi'
-	icon_state = "ppsh_bs"
-	item_state = "ppsh_bs"
-	origin_tech = list(TECH_COMBAT = 5, TECH_MATERIAL = 2)
-	price_tag = 800
-	serial_type = "NM"
-	excelsior = FALSE
-	gun_parts = list(/obj/item/part/gun/frame/ppsh = 1, /obj/item/part/gun/grip/rubber = 1, /obj/item/part/gun/mechanism/smg = 1, /obj/item/part/gun/barrel/pistol = 1)
-
-/obj/item/gun/projectile/automatic/ppsh/update_icon()
-	..()
-	if(ammo_magazine)
-		icon_state = "[initial(icon_state)]-full"
-		set_item_state("-full")
-	else
-		icon_state = initial(icon_state)
-		set_item_state()
-	return
-
-/obj/item/gun/projectile/automatic/ppsh/ppv
-	name = "\"Semyonovich\" assault SMG"
-	desc = "Some madmans bastardized yet idealized version of the past; improved yet its meaning and worth lost to time. \
-		 An experimental sub-machine gun design made for urban combat with a built in silencer and chambered in 9mm." // Don't add communist spiel on de-excelsiored guns for consistency, please.
-	icon = 'icons/obj/guns/projectile/ppv.dmi'
-	icon_state = "ppv"
-	item_state = "ppv"
-	mag_well = MAG_WELL_SMG|MAG_WELL_DRUM
-	origin_tech = list(TECH_COMBAT = 4, TECH_MATERIAL = 2)
-	price_tag = 800
-	damage_multiplier = 0.9
-	penetration_multiplier = 1
-	init_recoil = SMG_RECOIL(1.3)
-	auto_eject = 0
-	serial_type = "NM"
-	excelsior = FALSE
-	gun_parts = list(/obj/item/part/gun/frame/ppsh = 1, /obj/item/part/gun/grip/black = 1, /obj/item/part/gun/mechanism/smg = 1, /obj/item/part/gun/barrel/pistol = 1)
-
-/obj/item/gun/projectile/automatic/ppsh/ppv/update_icon()
-	..()
-	var/iconstring = initial(icon_state)
-	var/itemstring = ""
-
-	if (ammo_magazine)
-		iconstring += "[ammo_magazine? "_mag[ammo_magazine.max_ammo]": ""]"
-		itemstring += "_full"
-
-	if (!ammo_magazine || !length(ammo_magazine.stored_ammo))
-		iconstring += "_empty"
-
-	icon_state = iconstring
-	set_item_state(itemstring)
-
-/obj/item/part/gun/frame/ppsh
-	name = "PPSH frame"
-	desc = "A PPSH SMG frame. A worker's best friend."
-	icon_state = "frame_ppsh"
-	result = /obj/item/gun/projectile/automatic/ppsh
-	gripvars = list(/obj/item/part/gun/grip/excel, /obj/item/part/gun/grip/rubber, /obj/item/part/gun/grip/black)
-	mechanismvar = /obj/item/part/gun/mechanism/smg
-	barrelvars = list(/obj/item/part/gun/barrel/pistol)
-	resultvars = list(/obj/item/gun/projectile/automatic/ppsh, /obj/item/gun/projectile/automatic/ppsh/NM_colony, /obj/item/gun/projectile/automatic/ppsh/ppv)
+/obj/item/gun/projectile/automatic/ppsh
+	name = "Excelsior \"Shpagin\" assault SMG"
+	desc = "An outdated submachine gun made for urban assault teams. Chambered in 9mm - for the union, for the future."

```

Comment:
Personal preference suggestion: "Chambered in 9mm. For the union. For the future."